grails-app/domain/org/pih/warehouse/receiving/ReceiptItem.groovy

```diff
@@ -63,9 +63,13 @@ class ReceiptItem implements Comparable<ReceiptItem>, Serializable {
 		isSplitItem(nullable:true)
 		comment(nullable:true, maxSize: 255)
 	}
-	
+
+	def getReceivedQuantity() {
+		return receipt.receiptStatusCode == ReceiptStatusCode.RECEIVED ? quantityReceived : 0
```

Comment:
Having two getter methods to get received quantity seems like a recipe for disaster. Let's not do this.

Reply:
So remove this method and ...